src/pds/registrysweepers/ancestry/__init__.py

```diff
@@ -46,9 +47,24 @@ def run(
     nonaggregate_records = get_nonaggregate_ancestry_records(host, collection_records, registry_mock_query_f)
 
     ancestry_records = chain(bundle_records, collection_records, nonaggregate_records)
+    updates = generate_updates(ancestry_records, ancestry_records_accumulator, bulk_updates_sink)
+
+    if bulk_updates_sink is None:
+        log.info("Writing bulk updates to database...")
+        write_updated_docs(host, updates)
+    else:
+        list(updates)  # consume generator to dump bulk updates to sink
```

Comment:
Would it be more efficient (yeah, I know, premature optimization—root of all evil) to just say `del updates`?

Reply:
I didn't spare a second thought for it (given that it's a path only used with functional testing inputs and will always be trivial from a performance standpoint).

But the answer is "I have no idea - does `del` actually consume an iterator?".  I have no reason to think that it would.

An idiomatic efficient way would just be something like `for _ in updates: pass`.  I'll make that change if only so that it's more clear what's going on.

Reply:
[Done](https://github.com/NASA-PDS/registry-sweepers/pull/62/commits/b9d9eb86cca17a8530841d71eec9715ac9eb841e#diff-411a21299a42cb68599cc0b3a9314f8b07b52a1a3854cb7a1027c463b7fdcaaaR56-R58)

Reply:
@nutjob4life yielding (geddit?) to curiosity reveals that it doesn't

![image](https://github.com/NASA-PDS/registry-sweepers/assets/75815303/2df37a6b-6db2-45c9-a3d8-2eab60f2e912)

Reply:
Thanks @alexdunnjpl. (I sort of prefer the conciseness of `list(gen_me)` over the `for` loop—but I have terrible taste.)